Add person-team-relationship example (#148)
JupiterOne/docs: docs/person-team-relationship.md

# Mapping Person Entities to a Team

> 

Reviewer: Would be good to have a preamble describing why someone might want to do this.

Under the Assets section of the application, click the + button in the upper right corner.

> ![bulk-upload](../assets/bulk-upload.png)

Reviewer: Link also to the bulk upload API and suggest it can be done via UI or API

Reviewer 2: Is there *additional* API documentation for Bulk Upload other than https://support.jupiterone.io/hc/en-us/articles/360049348293?

> # Mapping Person Entities to a Team Entity

Reviewer 3: ```suggestion
# Creating `Team` entities and mapping team members
```

> This guide walks through the process of mapping Person(s) entities (already integrated via SSO/IdP providers) to a custom Team.

Reviewer 3: ```suggestion
This guide walks through the process of creating `Team` entities and mapping 
team members to the `Person` entities (that are already integrated via SSO/IdP 
providers) to each team.

This can be accomplished by uploading a JSON/YAML document containing the
`Team` entities either via the JupiterOne app UI or the platform API.
```

## UI 

Under the Assets section of the application, click the + button in the upper right corner.

![add-asset](../assets/add-asset.png)

Click the tile "Bulk upload from a JSON or YAML file".

![bulk-upload](../assets/bulk-upload.png)

 Use the following JSON as an example:

> ```

Reviewer 3: ```suggestion
```json
```

{
 "entities": [
   {
     "_key": "my_team:1",
     "_type": "my_team",
     "_class": "Team",
     "displayName": "Team One",
     "name": "Marketing Team",
     "description": "One person team",
     "email": "james@jupiterone.com",
     "members": [
       "james@jupiterone.com"
     ]
   },
   {
     "_key": "my_team:2",
     "_type": "my_team",
     "_class": "Team",
     "displayName": "Team Two",
     "name": "Solutions Team",
     "description": "Customer facing team",
     "email": "solutions@jupiterone.com",
     "lead": "cindy@jupiterone.com",

>      "members": [

Reviewer 3: It could be good to add explanation below the example code block to explain that:

- `"members"` are mapped to `Person` entities by email, creating `Team -HAS-> Person` relationships
- `"lead"` or `"manager"` or `"supervisor"` is also mapped to a `Person` entity by email, creating `Person -MANAGES-> Team` relationships

       "cindy@jupiterone.com",
       "james@jupiterone.com",
       "mark@jupiterone.com"
     ]
   }
 ]
}
```

## API

> This process can also be accomplished via the JupiterOne API using the JSON example above. See the document [Bulk Upload][bulk-upload-api] for more info.

Reviewer 3: please add hard line wrap for long lines
